components/support/utils/src/main/java/mozilla/components/support/utils/Browsers.kt

```diff
@@ -207,7 +208,12 @@ class Browsers private constructor(
         val intent = Intent(Intent.ACTION_VIEW)
         intent.data = uri
 
-        for (info in packageManager.queryIntentActivities(intent, 0)) {
+        val flag = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
+            PackageManager.MATCH_ALL
+        else
+            PackageManager.MATCH_DEFAULT_ONLY
```

Comment:
We require this to be wrapped in `{}` (See `detekt` failure).

Reply:
sorry, fixed ;)

Reply:
Thank you! Can you squash those commits so that this PR just contains one commit? :)

Reply:
Yep ;)